openpype/pipeline/colorspace.py

```diff
@@ -143,9 +151,20 @@ def get_colorspace_name_from_filepath(
     Returns:
         str: name of colorspace
     """
+    from .context_tools import get_current_context
+    context = get_current_context()
```

Comment:
I don't think we should support to use `get_current_context` at all. All of the arguments should be required.